packages/node_modules/@webex/react-component-utils/src/adaptiveCardUtil.js

```diff
@@ -108,12 +108,14 @@ export function getAdaptiveCard(
           window.open(action.url);
           break;
         case ADAPTIVE_CARD_SUBMIT_ACTION: {
-          const btnClicked = action.renderedElement;
-
-          btnClicked.disabled = true;
-          setTimeout(() => {
-            btnClicked.disabled = false;
-          }, 2500);
+          const btnClicked = action.renderedElement && action.parent.renderedElement;
```

Comment:
Could you add a comment as to why you're checking for the parent as well?

Reply:
@lalli-flores added the comment